tsconfig.json

```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "target": "es6",
+        "noImplicitAny": true,
+        "moduleResolution": "node",
+        "sourceMap": true,
+        "outDir": "functions",
```

Comment:
Cause it's how my tsconfigs were set up. So we'd set the functions dir to be like `functions/dist`?